.circleci/config.yml

```diff
@@ -26,6 +26,12 @@ variables:
       LANG: en_US.UTF-8
   linux: &linux
     machine: true
+  skip_fork: &skip_fork
```

Comment:
Nice! Please also add this to all other bulk and upload jobs.

Reply:
done

Reply:
I changed it so that `master`/`bulk` upload jobs fail on forks instead. The reason for this is that if a contributor tries to use their `master`/`bulk` for pull requests, we don't want it to silently succeed, but rather just fail and inform them to use another branch. For the nightly builds, I think it's nice to not let them fail and as such not bug the contributor with "failed build" notifications.

(Context for others: What we actually want is to be able to use `repo:branch` as a filter in the CircleCI `jobs`; but that is currently not supported. Hence this `skip_fork`/`fail_fork` workaround is the next best thing we can do if we still want to use `jobs`.)

Reply:
Looks like, as intended, the builds are terminated quickly after < 10 seconds:
https://circleci.com/gh/mbargull/bioconda-recipes/3
https://circleci.com/gh/mbargull/bioconda-recipes/4